src/sx-design/src/SxDesignProvider.js

```diff
@@ -98,7 +98,10 @@ export default function SxDesignProvider(props: Props): Node {
 
 /* eslint-disable sx/valid-usage */
 const styles = sx.create({
-  common: SxDesignProviderCSSVariables.common,
+  common: {
+    ...SxDesignProviderCSSVariables.common,
+    fontFamily: 'sans-serif',
```

Comment:
How about `ui-sans-serif`?

Reply:
I've changed it to `ui-sans-serif, sans-serif` because only `ui-sans-serif` is widely unsupported: https://caniuse.com/extended-system-fonts

Reply:
In that case, maybe `ui-sans-serif, system-ui, sans-serif`?

Reply:
https://caniuse.com/?search=system-ui

Reply:
Your comments motivated me to dig a bit deeper into this problem and I choose to copy `font-family` used here on GitHub.com. It looks the best from what I tried. Hopefully, this is the best what we can do now. Please, have a look again.